src/CFixer/main.cpp

```diff
@@ -14,7 +14,9 @@ using namespace CopyrighFixer;
 
 int main(int argc, char *argv[]) {
 
-    Worker worker;
+    Worker worker(new Config,
+                  new ConfigParser,
+                  new Signer);
```

Comment:
Constructor of the Worker class must be initialized without arguments.
```suggestion
    Worker worker;
```